Objects/Converters/ConverterAutocadCivil/ConverterAutocadCivilShared/ConverterAutocadCivil.Civil.cs

```diff
@@ -505,6 +542,97 @@ private Arc ProfileArcToSpeckle(CivilDB.ProfileCircular circular)
       return LineToSpeckle(new LineSegment2d(start, end));
     }
 
+    /*
```

Comment:
Do we need this huge comment chunk here though? :) If it can be removed it would be nice!

I'll leave the decision to you though